department-of-veterans-affairs/va.gov-cms
# VACMS-8149: Add 2:3 crop style and update 2:3 thumbnail style.

## Description

Closes #8149.

## Testing done


## Screenshots
![image](https://user-images.githubusercontent.com/8375335/155614641-d8cf4bfa-138f-43dd-9fdc-a7c749728709.png)
![image](https://user-images.githubusercontent.com/8375335/155614843-93c17dbc-4d94-4afb-ab97-f822f8a13918.png)


## QA steps

As a cms editor
1. I can see and select the crop type 2:3 when inserting media
2. I can confirm on the frontend that the 2:3 crop is applied to media


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
